ontology/scdo.md

```diff
@@ -19,6 +19,19 @@ homepage: https://scdontology.h3abionet.org/
 products:
   - id: scdo.owl
   - id: scdo.obo
+publications: 
+   - id:  https://doi.org/10.1093/database/baac014
+     title: "The Sickle Cell Disease Ontology: recent development and expansion of the universal sickle cell knowledge representation."
+   - id: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7549008/
+     title: "The Sickle Cell Disease Ontology: Enabling Collaborative Research and Co-Designing of New Planetary Health Applications."
+   - id: https://academic.oup.com/database/article/doi/10.1093/database/baz118/5626537 
+     title: "The Sickle Cell Disease Ontology: enabling universal sickle cell-based knowledge representation."
+tracker: https://github.com/scdodev/scdo-ontology/issues
+license:
+  url: https://www.gnu.org/licenses/gpl-3.0.en.html
+  label: GPL-3.0
```

Comment:
any change you can change this to an appropriate license for data from Creative Commons like CC BY 4.0?

Reply:
I will check and get back to you.